controllers/activemqartemis_reconciler.go

```diff
@@ -301,6 +303,10 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) ProcessStatefulSet(customResour
 }
 
 func isClustered(customResource *brokerv1beta1.ActiveMQArtemis) bool {
+	if isRestricted(customResource) {
```

Comment:
I think that will have to be documented or will flow from things like existing RBAC roles and the ability to work without passwords. I think it will be easier to demonstrate/document the subset that works through examples and tests. In essence what is left will be configured through brokerProperties, as a way to open up the broker.

Reply:
I added some doc to mark this feature experimental and provide some high level detail on what it does. By experimenting we will be able to understand the ways in which this mode of operation needs to be extended.